build-tools/packages/build-cli/docs/bump.md

```diff
@@ -70,12 +70,13 @@ Update the dependency version of a specified package or release group. That is,
 USAGE
   $ flub bump deps PACKAGE_OR_RELEASE_GROUP [-v] [--prerelease -t
     latest|newest|greatest|minor|patch|@next|@canary] [--onlyBumpPrerelease] [-g
-    client|server|azure|build-tools|gitrest|historian | -p <value>] [-x | --install | --commit |  |  | ]
+    client|server|azure|build-tools|gitrest|historian | -p <value>] [-x | --install | --commit |  |  | ] [-b <value>]
 
 ARGUMENTS
   PACKAGE_OR_RELEASE_GROUP  The name of a package or a release group.
 
 FLAGS
+  -b, --branchName=<value>     Takes in the branch name as main or next
```

Comment:
Are we forcing them to call out branch here? I would imagine user would already be on the particular branch by the time they are executing this command?

Reply:
Yes -- this is forcing them to call out the branch explicitly just in case of bumping prereleases. The command would look like:

`flub bump deps server -g client -t latest -b next --prerelease`

Reply:
Removed the `branchName` flag. Modified the code to fetch the current branch name.